### ISSUE TITLE
HTMS Potential in Transfer Filters

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1208]
Original [issue 1208](https://code.google.com/p/foxtrick/issues/detail?id=1208) created by [tasosventouris](mailto:tasosventouris@gmail.com) on 2014-06-19T16:39:06.000Z:

**From:** eka018
**PostID:** [(16412248.166)](https://www.hattrick.org/goto.ashx?path=/Forum/Read.aspx%3Ft%3D16412248%26n%3D166%26v%3D4)
**Reply:** [(16412248.1)](https://www.hattrick.org/goto.ashx?path=/Forum/Read.aspx%3Ft%3D16412248%26n%3D1%26v%3D4)
**To:** Everyone
**Datetime:** 19/6/2014 1at 3:47
**Message:** 
Sorry, I posted in the wrong thread:[post=16497555.151]

Short version: An option to choose f.ex. minimum 2000 HTMS-potensial when your search for players. That would be really great! Having to go through tons of crappy talents can be avoided with this change.

[hr]
[q]Possible spoiler.[u]Click here to show[/u] [/q]
